games-fps/fteqw/fteqw-20240406~git.recipe

```diff
@@ -0,0 +1,102 @@
+SUMMARY="A highly versatile game engine originally based on QuakeWorld"
+DESCRIPTION="FTEQW is a highly versatile game engine originally based on QuakeWorld.
+
+Features:
+
+- Single & multi-player support.
+- Can run games on the Quake, Quake II, and Quake III engines.
+- Vast amount of map, model & image formats are supported.
+- Advanced console, with descriptions & autocompletion.
+- Plugin support, enabling use of FFMPEG, Bullet/ODE physics & more.
+- Extensive suite of QuakeC/entity debugging features.
+- Deep integration with FTEQCC, which can even be executed in-game.
+- Support for split-screen local multiplayer.
+- Voice-chat via Opus & Speex.
+- Support for hundreds of players on a single server.
+- Works on Windows, Linux, OpenBSD, Haiku... & more!
+- New features are added all the time in cooperation with modders!"
+HOMEPAGE="https://www.fteqw.org/"
+COPYRIGHT="2002-2023 Forethought Entertainment"
+LICENSE="GNU GPL v2"
+REVISION="1"
+gitRevision="1f9f3635f0aef3b2eed6b40e35fcf6223c6ad533"
+SOURCE_URI="https://github.com/fte-team/fteqw/archive/$gitRevision.tar.gz"
+CHECKSUM_SHA256="c9376b7516444cc12e04a771f6e3bad91146762c3ecadc2aeda87287922dc48e"
+SOURCE_DIR="fteqw-$gitRevision"
+ADDITIONAL_FILES="fteqw-$portVersion.rdef.in"
+PATCHES="fteqw-$portVersion.patchset"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	fteqw$secondaryArchSuffix = $portVersion
+	cmd:fteqw
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libSDL2_2.0$secondaryArchSuffix
+	lib:libvorbis$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libgnutls$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+	lib:libbz2$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libGL$secondaryArchSuffix
+	devel:libSDL2_2.0$secondaryArchSuffix
+	devel:libvorbis$secondaryArchSuffix
+	devel:libfreetype$secondaryArchSuffix
+	devel:libgnutls$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	devel:libpng16$secondaryArchSuffix
+	devel:libbz2$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:strip
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	cmake -Bbuild -S. \
+		-DCMAKE_BUILD_TYPE=Release \
+		-DFTE_USE_SDL=true \
+		-DFTE_LIBS=network
+	make -Cbuild fteqw $jobArgs
+	strip build/fteqw
+}
+
+INSTALL()
+{
+	mkdir -p $binDir
+	cp -a build/fteqw $binDir/fteqw
```

Comment:
As mentioned on IRC, best to put this in $prefix/bin instead of $binDir for 32bit. (same for adding the resources to the binary).